cli/winresources/docker-proxy/winresources.go

```diff
@@ -0,0 +1,11 @@
+// Package winresources is used to embed Windows resources into docker-proxy.exe.
+//
+// These resources are used to provide:
+// * Version information
+// * An icon
+// * A Windows manifest declaring Windows version support
+//
+// The resource object files are generated when building with goversioninfo
+// in hack/make/.go-autogen and are located in cmd/docker-proxy/winresources.
+// This occurs automatically when you cross build against Windows OS.
+package winresources
```

Comment:
I wonder if instead of `cli/winresources/docker-proxy` it could be put in `cmd/docker-proxy/winresources`? That way it's closer to the thing it's specific to. In fact, it probably wouldn't need to be a subdirectory, what if the syso is output directly into `cmd/docker-proxy`? If as I suggested elsewhere it gets the right suffixes, then it won't interfere with any cross-builds etc either.

(Where `docker-proxy` means the same thing for `dockerd`.)

Actually, I just noticed that the comment _says_ it does what I'm suggesting, but the `go:generate` line is outputting into `cli/winresources` where this file currently lives. Was a restructuring done away from `cmd/docker-proxy/winresources` to `cli/winresources/docker-proxy` at some point?

Reply:
We are mounting a tmpfs for `cli/winresources/docker-proxy` and `cli/winresources/dockerd` in our Dockerfile for generated resources (syso). If we move to `cmd/docker-proxy/winresources` or `cmd/dockerd/winresources` the stub package would be removed in there as well as manifest and icon. That why it's splitted that way. We could do that if we bind mount the working tree as `rw` instead of `ro` (default) but don't think it's a good idea.